### DISCUSSION TITLE
Does GitHub Pages support folders starting with a dot?

### DISCUSSION THREAD

**Discussion by markvincze** [community/community#22227] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>I’d like to ask a question related to GH Pages.</p>
<p>I migrated my blog from Ghost to a statically generated sit, and I’m trying to deploy it with GH pages.</p>
<p>I have some routes in my blog where a route segment (thus the name of the corresponding folder starts with a dot).</p>
<p>For example the content for the page of the <code>.net</code> tag on my blog is here: <a href="https://github.com/markvincze/blog/tree/gh-pages/tags/.net" rel="noopener nofollow ugc">blog/tags/.net at gh-pages · markvincze/blog (github.com)</a>, but on the site it doesn’t work, it displays the GH Pages 404 screen: <a href="https://blog.markvincze.com/tags/.net" rel="noopener nofollow ugc">Page not found · GitHub Pages (markvincze.com)</a><br>
(The other tag pages, which don’t start with a dot, work properly.)</p>
<p>Is this not supported? Or should the <code>.</code> in the folder name be escaped with something else?</p>
<p>Thanks!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/markvincze">@markvincze</a>, welcome to the GitHub Support Community! It’s probably Jekyll that’s blocking that folder from being included in your deploy. Jekyll is run be default and ignores folders that start with certain characters.</p>
<p>It looks like you’re not using Jekyll so you can disable it by creating an empty file called <code>.nojekyll</code> in the root of your <code>gh-pages</code> branch. Once you’ve done that any folders starting with <code>.</code> should be included in your site build.</p>

**Reply by TomasHubelbauer** [reply]
In case you are using Jekyll and would like to expose some implicitly excluded paths, append `include: ['.something']` to your Jekyll `_config.yml` file.

**Reply by ooker777** [reply]
[Bypassing Jekyll on GitHub Pages](https://github.blog/news-insights/bypassing-jekyll-on-github-pages/ "Bypassing Jekyll on GitHub Pages - The GitHub Blog")

**Comment by markvincze** [comment]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a>,</p>
<p>This fixed the problem, thanks a bunch!</p>

**Comment by mvgfr** [comment]
FWIW: adding a ".nojekyll" file still works, for allowing dot-files - thx
